include/grpc/grpc.h

```diff
@@ -427,6 +436,8 @@ void grpc_channel_watch_connectivity_state(
     completions are sent to 'completion_queue'. 'method' and 'host' need only
     live through the invocation of this function. */
 grpc_call *grpc_channel_create_call(grpc_channel *channel,
+                                    grpc_call *parent_call,
+                                    gpr_uint32 inheritance_mask,
```

Comment:
I'm not going to disallow it (I think)... provided GRPC_PROPAGATE_DEFAULTS as the 'recommended' set - will document as such shortly - so that you don't accidentally pick up additional things depending on which libgrpc.so is on a system.